expression/builtin/info.go

```diff
@@ -76,3 +76,12 @@ func builtinConnectionID(args []interface{}, data map[interface{}]interface{}) (
 	ctx := c.(context.Context)
 	return variable.GetSessionVars(ctx).ConnectionID, nil
 }
+
+func builtinVersion(args []interface{}, data map[interface{}]interface{}) (v interface{}, err error) {
+	c, ok := data[ExprEvalArgCtx]
+	if !ok {
+		return nil, errors.Errorf("Missing ExprEvalArgCtx when evalue builtin")
+	}
+	ctx := c.(context.Context)
+	return variable.GetSessionVars(ctx).Version, nil
```

Comment:
Version is not a session variable. So I think it is better to return mysql.ServerVersion here.